hedera-node/docs/design/services/smart-contract-service/system-contract-abi-guidelines.md

```diff
@@ -0,0 +1,132 @@
+# System Contract API Design Guidelines
```

Comment:
nit: I would consider sticking to either `API` or `ABI` throughout the document (where possible) to avoid any ambiguity 🤔

Reply:
Oh darn it.  You're right of course.